lib/routes/tongji/sem/_utils.ts

```diff
@@ -0,0 +1,69 @@
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+
+export async function getNotifByPage(pageNumber: number) {
+    const pageUrl: string = `https://sem.tongji.edu.cn/semch/category/frontpage/notice/page/${pageNumber}`;
+
+    try {
+        const response = await got.get(pageUrl, {
+            headers: {
+                'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/38.0.101.76 Safari/537.36',
```

Comment:
RSSHub will use a randomised user agent of [Chrome on mac](https://github.com/DIYgod/RSSHub/blob/263d6b818ea551e9a498207ba1f0a912b6370224/lib/config.ts#L344) by default. Does the site only works with this fixed old version of Chrome on Windows?